src/System.Drawing.Common/tests/FontTests.cs

```diff
@@ -835,7 +834,7 @@ public void ToLogFont_DisposedGraphics_ThrowsArgumentException()
             }
         }
 
-        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Auto)]
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
```

Comment:
@safern, @jkoritzinsky, might be worth a quick audit to see if this same situation occurs elsewhere in the library.  This is the only one we hit in tests, but our tests aren't stellar.

Reply:
Makes sense... https://github.com/dotnet/corefx/issues/36722